README.md

```diff
@@ -273,6 +290,32 @@ associative array in the first argument:
 </div>
 ```
 
+You can also pass parameters to actions:
+
+```twig
+<div {{ stimulus_action('controller', 'method', 'click', { 'count': 3 }) }}>Hello</div>
```

Comment:
```suggestion
<div {{ stimulus_action('hello-controller', 'method', 'click', { 'count': 3 }) }}>Hello</div>
```

... just to be clear that the first arg is your controller name, not the specific word `controller`.